test/api/test_gpu.cpp

```diff
@@ -49,6 +49,42 @@ TEST_CASE(load_and_run)
     CHECK(bool{shapes_before.front() == outputs.front().get_shape()});
 }
 
+TEST_CASE(load_and_run_async)
+{
+    auto p             = migraphx::parse_onnx("conv_relu_maxpool_test.onnx");
+    auto shapes_before = p.get_output_shapes();
+    migraphx::compile_options options;
+    options.set_offload_copy();
```

Comment:
We dont want offload copy set as that will copy back and forth from the host.

Reply:
So we should be binding memory locations for that? Similar examples of that anywhere?